integtest/src/main/java/org/corfudb/universe/cluster/docker/DockerCluster.java

```diff
@@ -0,0 +1,137 @@
+package org.corfudb.universe.cluster.docker;
+
+import com.google.common.collect.ImmutableList;
+import com.spotify.docker.client.DockerClient;
+import com.spotify.docker.client.exceptions.DockerException;
+import com.spotify.docker.client.messages.NetworkConfig;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.universe.cluster.Cluster;
+import org.corfudb.universe.cluster.ClusterException;
+import org.corfudb.universe.service.DockerService;
+import org.corfudb.universe.service.Service;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.stream.Collectors'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)